packages/phrases/src/locales/en.ts

```diff
@@ -413,6 +415,13 @@ const translation = {
         no_connector:
           'You haven’t set up any social connectors yet. Your sign in experience won’t go live until you finish the settings first. ',
       },
+      save_alert: {
+        title: 'Reminder',
+        description:
+          'You are changing sign in methods from one to another. This will impact some of your users. Are you sure you want to do that?',
```

Comment:
nit: better remove `want to`?

Reply:
Copied from Figma, we have issue to track the phasse alignment. So I'll leave it as it is now.